server/portal/apps/tickets/api/views.py

```diff
@@ -70,14 +72,26 @@ def post(self, request):
 
         problem_description += "\n\n" + metadata
 
-        ticket_id = rt.create_ticket(subject=subject,
-                                     problem_description=problem_description,
-                                     requestor=email,
-                                     cc=cc,
-                                     attachments=attachments)
+        recap_util = utils.DjangoRecaptcha()
 
-        return JsonResponse({'ticket_id': ticket_id})
+        if not request.user.is_authenticated:
+            recap_result = recap_util.get_recaptcha_verification(request)
+            if recap_result['success']:
+                ticket_id = rt.create_ticket(subject=subject,
+                                         problem_description=problem_description,
+                                         requestor=email,
+                                         cc=cc,
+                                         attachments=attachments)
+                return JsonResponse({'ticket_id': ticket_id})
+            else:
```

Comment:
Since the ticket create code is the same, we could simplify this
```suggestion
            if not recap_result['success']:
```